rllib/execution/rollout_ops.py

```diff
@@ -337,6 +337,35 @@ def __call__(self, samples: SampleBatchType) -> SampleBatchType:
         return samples
 
 
+def standardize_fields(samples: SampleBatchType, fields: List[str]) -> SampleBatchType:
+    """Standardize fields of the given SampleBatch"""
```

Comment:
Actually, could we just simplify this even further and directly add the following code to the `training_iteration()` method:

```
        # Standardize `advantages` values in train_batch.
        for policy_id, batch in train_batch.policy_batches.items():
            if Postprocessing.ADVANTAGES not in batch:
                raise KeyError(
                    f"`{Postprocessing.ADVANTAGES}` not found in SampleBatch for "
                    f"policy `{policy_id}`! Maybe this policy fails to add "
                    f"`{Postprocessing.ADVANTAGES}` in its `postprocess_trajectory` "
                    f"method? Or this policy is not meant to learn at all and you "
                    "forgot to remove it from the list under `config."
                    "multiagent.policies_to_train`."
                )
            batch[Postprocessing.ADVANTAGES] = standardized(
                batch[Postprocessing.ADVANTAGES])

```

Reply:
This would require two identical for loops, which are quite ugly IMO. This is because we need to train AFTER normalizing, not before.

Reply:
Ok, fair enough. Fine with me :)